File: app/src/main/java/com/warchaser/daggertest/utils/ImageUtils.java
package com.warchaser.daggertest.utils;

public class ImageUtils {

    public static final String BASE_POSTER_PATH = "http://image.tmdb.org/t/p/w342";

    private ImageUtils(){

    }

    public static String getPosterUri(String posterPath){
        return BASE_POSTER_PATH + posterPath;
    }

}

File: app/src/main/java/com/warchaser/daggertest/utils/ImageUtils.kt
package com.warchaser.daggertest.utils;

object ImageUtils {

    private const val BASE_POSTER_PATH : String = "http://image.tmdb.org/t/p/w342"

    @JvmStatic
    fun getPosterUri(posterPath : String) : String{
        return BASE_POSTER_PATH + posterPath
    }

}
